paasta_tools/api/api.py

```diff
@@ -148,6 +156,9 @@ def main(argv=None):
     if args.soa_dir:
         os.environ["PAASTA_API_SOA_DIR"] = args.soa_dir
 
+    if args.cluster:
```

Comment:
Do you need all three? Can you just do the CLI arg instead of having the cli/config/and env ?

Reply:
settings.cluster is the one that needs to be here. I put env here because I don't want to pass "args" to the setup_paasta_api function or make it global. Also I want to be more consistent so it's easier to read.